xla/service/gpu/dot_algorithm_support_test.cc

```diff
@@ -59,35 +63,38 @@ struct Sizes {
 struct TestParams {
   using TupleType =
       std::tuple<PrecisionConfig::Algorithm, PrimitiveType, PrimitiveType,
-                 se::CudaComputeCapability, BackendRestriction, Sizes>;
+                 se::CudaComputeCapability, BackendRestriction, Sizes, int32_t>;
 
   PrecisionConfig::Algorithm algorithm;
   PrimitiveType input_storage_type;
   PrimitiveType output_storage_type;
   se::CudaComputeCapability min_cuda_capability;
   BackendRestriction backend_restriction;
   Sizes sizes;
+  int32_t min_rocm_version;
 
   explicit TestParams(TupleType t)
       : algorithm(std::get<0>(t)),
         input_storage_type(std::get<1>(t)),
         output_storage_type(std::get<2>(t)),
         min_cuda_capability(std::get<3>(t)),
         backend_restriction(std::get<4>(t)),
-        sizes(std::get<5>(t)) {}
+        sizes(std::get<5>(t)),
+        min_rocm_version(std::get<6>(t)) {}
 };
 
 std::string TestParamsToString(
     const TestParamInfo<TestParams::TupleType>& info) {
   const TestParams params(info.param);
   return absl::StrFormat(
-      "%s_with_input_%s_output_%s_from_cc_%d_%d_%s_c_%d_nc_%d",
+      "%s_with_input_%s_output_%s_from_cc_%d_%d_%s_c_%d_nc_%d_rocm_%d",
```

Comment:
Nit: the rocm version kindof belongs next to the cuda version.
Something like this:
%s_with_input_%s_output_%s_from_cc_%d_%d_rocm_%d_%s_c_%d_nc_%d